apps/rush-lib/src/cli/scriptActions/BulkScriptAction.ts

```diff
@@ -182,37 +203,76 @@ export class BulkScriptAction extends BaseScriptAction {
-        'to specify the project in the current working directory.',
+        'Run command on the selection instead of all projects. ' +
+        'Add the specified project and all of its direct or indirect dependencies to the current selection. ' +
+        '"." can be used as shorthand to specify the project in the current working directory. ' +
+        'Additional use of "--from" or "--to" will further expand the selection.',
       completions: this._getProjectNames.bind(this)
     });
-    this._fromVersionPolicy = this.defineStringListParameter({
-      parameterLongName: '--from-version-policy',
-      argumentName: 'VERSION_POLICY_NAME',
+    this._fromExceptProject = this.defineStringListParameter({
+      parameterLongName: '--from-except',
+      parameterShortName: '-F',
+      argumentName: 'PROJECT1',
       description:
-        'Run command in all projects with the specified version policy ' +
-        'and all projects that directly or indirectly depend on projects with the specified version policy'
+        'Run command on the selection instead of all projects. ' +
+        'Add all projects that depend on the specified project (excluding itself) to the current selection. ' +
+        '"." can be used as shorthand to specify the project in the current working directory. ' +
+        'Additional use of "--from" or "--to" will further expand the selection.',
+      completions: this._getProjectNames.bind(this)
     });
+
     this._toVersionPolicy = this.defineStringListParameter({
       parameterLongName: '--to-version-policy',
       argumentName: 'VERSION_POLICY_NAME',
       description:
-        'Run command in all projects with the specified version policy and all of their dependencies'
+        'Run command on the selection instead of all projects. ' +
+        'Adds all projects with the specified version policy, and all dependencies thereof, to the current selection.'
     });
-    this._fromFlag = this.defineStringListParameter({
-      parameterLongName: '--from',
-      parameterShortName: '-f',
-      argumentName: 'PROJECT2',
+    this._fromVersionPolicy = this.defineStringListParameter({
+      parameterLongName: '--from-version-policy',
+      argumentName: 'VERSION_POLICY_NAME',
       description:
-        'Run command in the specified project and all projects that directly or indirectly depend on the ' +
-        'specified project. "." can be used as shorthand to specify the project in the current working directory.',
-      completions: this._getProjectNames.bind(this)
+        'Run command on the selection instead of all projects. ' +
+        'Adds all projects with the specified version policy, and all projects that depend on them, to the current selection.'
     });
+
+    this._withDependencies = this.defineFlagParameter({
+      parameterLongName: '--with-dependencies',
+      parameterShortName: '-w',
```

Comment:
It occurs to me that we may want to reserve `-w` for `--watch`, unless we would prefer for watching to be a completely different command.

Reply:
Agreed!